static/app/views/starfish/views/screens/screenLoadSpans/samples/samplesContainer.tsx

```diff
@@ -0,0 +1,133 @@
+import {Fragment, useCallback, useState} from 'react';
+import styled from '@emotion/styled';
+import debounce from 'lodash/debounce';
+
+import {COL_WIDTH_UNDEFINED} from 'sentry/components/gridEditable';
+import {space} from 'sentry/styles/space';
+import useRouter from 'sentry/utils/useRouter';
+import {DurationCell} from 'sentry/views/starfish/components/tableCells/durationCell';
+import {
+  SpanSummaryQueryFilters,
+  useSpanMetrics,
+} from 'sentry/views/starfish/queries/useSpanMetrics';
+import {SpanMetricsField} from 'sentry/views/starfish/types';
+import {DataTitles} from 'sentry/views/starfish/views/spans/types';
+import {Block} from 'sentry/views/starfish/views/spanSummaryPage/block';
+import DurationChart from 'sentry/views/starfish/views/spanSummaryPage/sampleList/durationChart';
+import SampleTable from 'sentry/views/starfish/views/spanSummaryPage/sampleList/sampleTable/sampleTable';
+
+const {SPAN_SELF_TIME, SPAN_OP} = SpanMetricsField;
+
+type Props = {
+  groupId: string;
+  transactionName: string;
+  release?: string;
+  sectionSubtitle?: string;
+  sectionTitle?: string;
+  transactionMethod?: string;
+};
+
+export function ScreenLoadSampleContainer({
+  groupId,
+  transactionName,
+  transactionMethod,
+  sectionTitle,
+  release,
+}: Props) {
+  const router = useRouter();
+  const [highlightedSpanId, setHighlightedSpanId] = useState<string | undefined>(
+    undefined
+  );
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const debounceSetHighlightedSpanId = useCallback(
+    debounce(id => {
+      setHighlightedSpanId(id);
+    }, 10),
+    []
+  );
+
+  const filters: SpanSummaryQueryFilters = {
+    transactionName,
+  };
+
+  if (transactionMethod) {
+    filters['transaction.method'] = transactionMethod;
+  }
+
+  if (release) {
+    filters.release = release;
+  }
+
+  const {data: spanMetrics} = useSpanMetrics(
+    groupId,
+    filters,
+    [`avg(${SPAN_SELF_TIME})`, SPAN_OP],
+    'api.starfish.span-summary-panel-samples-table-avg'
+  );
+
+  return (
+    <Fragment>
+      {sectionTitle && <SectionTitle>{sectionTitle}</SectionTitle>}
+      {release && <SectionSubtitle>{release}</SectionSubtitle>}
```

Comment:
Should we link to release page from here?

Reply:
I am working on a PR that cleans up release display & info through the entire screen load flow, so I'll address this as a part of that!